R/pkg/R/DataFrame.R

```diff
@@ -946,15 +946,36 @@ setMethod("foreachPartition",
 
 ############################## SELECT ##################################
 
-getColumn <- function(x, c) {
-  column(callJMethod(x@sdf, "col", c))
-}
+#' Select a column
+#'
+#' Selects column based on the column name and return it as a Column.
+#' Note that the column name can also reference to a nested column like `a.b`.
+#'
+#' @param x A SparkSQL DataFrame
+#'
+#' @rdname select
+#' @name col
+#' @aliases col
+#' @export
+#' @examples
+#'\dontrun{
+#' sc <- sparkR.init()
+#' sqlContext <- sparkRSQL.init(sc)
+#' df <- createDataFrame(sqlContext, list(50, 60, 70), "age")
+#' col(df, "age")
+#'}
+setMethod("col",
```

Comment:
col is a API method of Scala DataFrame, my intention to add it in SparkR is for API parity. But if it conflicts with existing function in R base package, and we have alias functions ($ and [[]]), I agree that we can roll such change back.